_posts/2019-12-15-phone-or-laptop.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Phone, Tablet or Laptop as Daily Driver
+date: 2019-12-15 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author:
+---
+
+<!-- 2014-09-09 -->
+
+[The Tablet Loses](https://brooksreview.net/2014/09/the-tablet-loses/) :
+>If you could only buy one device out of a laptop, tablet, or phone which would you buy? If it is me I’m buying the phone. It would be annoying, certainly, but my iPhone can do everything my iPad can do, and my iPad can do almost everything my Mac can do. So, there’s that.
+
+I'm not on the same boat at all.
+
+If I'd have to choose between my Mac and my iPhone I'd choose my Mac, hands down. The iPhone is just way too limited. Too limited to read (books, and long articles), too limited to write on (as before novels or long articles), too limited to work on (no Xcode, no easy to use terminal, ftp, ssh, no good filesystem). Okay, I don't have an external keyboard for it and with one it might be easier to type on it, but who in their right minds would have one for an iPhone? How would you be able to read what you write from so far away? On an iPad it is different and if you're a manager, a writer or a podcaster it might be enough for you. For programmers it is not and even if Apple will one day add Xcode to it which does not seem like it is happening soon most programmers I know will still use a mac instead.
```

Comment:
minds - > mind

Reply:
manager - - what's the space there for

Reply:
which does not seem like it is happening soon - > put in brackets

Reply:
Second programmers is useless

Reply:
> manager - - what's the space there for

Not sure what you mean with that..?

Reply:
Rest is fixed

Reply:
> > manager - - what's the space there for
> 
> Not sure what you mean with that..?

@MsSlacker ^